README.md

```diff
@@ -48,19 +48,49 @@ class AppKernel extends Kernel
 }
 ```
 
-Step 3: Configuration
----------------------
+Configure services
+------------------
+
+This bundle allows to easily configure the caching service the extension will use for the caching. The extension 
+by default supports instances of `Doctrine\Common\Cache\Cache` but allows you to use packages that 
+provide a `psr/cache-implementation`.
 
 ```yml
 # app/config/config.yml
 twig_cache:
-    service: cache_service # instance of Doctrine\Common\Cache\Cache
+    service: cache_service # 
```

Comment:
While waiting for `asm89/twig-cache-extension:v1.3.0`, could you please add a description of allowed classes here or remove the `#` too?